src/date-input/index.tsx

```diff
@@ -133,17 +132,13 @@ export default factory(function({ properties, middleware: { theme, icache, i18n,
 										}
 									}}
 									trailing={() => (
-										<Button
+										<button
```

Comment:
can you add `type='button'` please so stop submits and in the `onclick` you'll need to prevent default etc on the event and ensure it is not leaked to the `openCalendar` callback